patches/minecraft/net/minecraft/client/gui/inventory/GuiContainerCreative.java.patch

```diff
@@ -67,15 +67,19 @@
          if (this.field_147062_A.func_146179_b().isEmpty())
          {
              for (Item item : Item.field_150901_e)
-@@ -366,7 +403,7 @@
+@@ -366,10 +403,10 @@
      {
          CreativeTabs creativetabs = CreativeTabs.field_78032_a[field_147058_w];
  
 -        if (creativetabs.func_78019_g())
 +        if (creativetabs != null && creativetabs.func_78019_g())
          {
              GlStateManager.func_179084_k();
-             this.field_146289_q.func_78276_b(I18n.func_135052_a(creativetabs.func_78024_c()), 8, 6, 4210752);
```

Comment:
A question: Wouldn't it be possible to override func_78024_c()? (or to make it protected, so that it can be overridden?) This would reduce patch size to 1 or 2 lines (depending on whether you insert a Nullability check, or not...)...
Or am I missing something?